docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3.4"
+services:
+  docker:
+    image: homework25:latest
+    container_name: docker-compose
+    build:
```

Comment:
do we need to use `image:` and `build:` at the same time :thinking:

Reply:
today I just studied 26 homework, and saw when using image, this is just already a manual assembly, then build is not needed